api/migrations/versions/5d5b0524b4fa_add_parent_id_to_response.py

```diff
@@ -0,0 +1,28 @@
+"""empty message
+
+Revision ID: 5d5b0524b4fa
+Revises: 627a96dad7e8
+Create Date: 2023-05-27 07:46:16.735763
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '5d5b0524b4fa'
+down_revision = '627a96dad7e8'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('response', sa.Column('parent_id', sa.Integer(), nullable=True))
+    op.create_foreign_key(None, 'response', 'response', ['parent_id'], ['id'])
```

Comment:
Please add a name to the foreign key, otherwise it can create some issues later.